### PR TITLE
api initial routes

### PR DESCRIPTION
**Pull request by TomaCode**

### CODE REVIEW

File: src/TourManager/Api/Controllers/TourController.cs [hunk 1]
```diff
@@ -4,13 +4,47 @@
 namespace Travely.TourManager.Api.Controllers
 {
     [ApiController]
-    [Route("[controller]")]
+    [Route("api/v1/[controller]")]
```
Comment (HAghajanyan-ST) [inline src/TourManager/Api/Controllers/TourController.cs:7]: I will suggest to use **Microsoft.AspNetCore.Mvc.Versioning** for versioning

File: src/TourManager/Api/Controllers/TourController.cs [hunk 2]
```diff
@@ -4,13 +4,47 @@
...
     public class TourController : Controller
     {
+        // GET: api/v1/tour
         [HttpGet]
+        [Route("")]
```
Comment (HAghajanyan-ST) [inline src/TourManager/Api/Controllers/TourController.cs:12]: Remove empty route attribute

File: src/TourManager/Api/Controllers/TourController.cs [hunk 3]
```diff
@@ -4,13 +4,47 @@
...
         public async Task<IActionResult> Get()
         {
             return Ok();
         }
+
+        // GET: api/v1/tour/{id}
+        [HttpGet]
+        [Route("{id}")]
```
Comment (HAghajanyan-ST) [inline src/TourManager/Api/Controllers/TourController.cs:20]: You can specify route parameters in HttpGet attribute

**Review by HAghajanyan-ST — CHANGES_REQUESTED** [review]

File: src/TourManager/Api/Controllers/TourController.cs (hunk 3 above)
Comment (TomaCode) [reply src/TourManager/Api/Controllers/TourController.cs]: I'm not sure the latest changes are applicable for post/put/delete requests. have a look please.

**Pull request closed without merging**